### PR TITLE
1451 docketrecord sortfuncs

### PR DESCRIPTION
**Pull request by kkoskelin**
sort functions, tests

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedCaseDetail.js [hunk 1]
```diff
@@ -207,13 +208,62 @@ const formatCase = (caseDetail, caseDetailErrors) => {
...
+    // reversing AFTER the sort keeps sorting stable
+    return result.reverse();
+  }
+  return result;
+};
+
 export const formattedCases = get => {
   const cases = get(state.cases);
-  return cases.map(formatCase);
+  const docketRecordSort = get(state.sessionMetadata.docketRecordSort);
```
Comment (wbyoko) [inline web-client/src/presenter/computeds/formattedCaseDetail.js:248]: isn't this supposed to be per case, and not general

Comment (rachaelparris) [reply web-client/src/presenter/computeds/formattedCaseDetail.js:248]: I think we should merge this and figure out how to do that on my views branch.

Comment (kkoskelin) [reply web-client/src/presenter/computeds/formattedCaseDetail.js:248]: That may be the case -- I'd allow @rachaelparris to tweak the calls into state which retrieve the method of sorting in her PR?

Comment (wbyoko) [reply web-client/src/presenter/computeds/formattedCaseDetail.js:248]: Okay. Sounds good to me.

**Pull request merged**